src/encord_active/lib/model_predictions/prediction_writer.py

```diff
@@ -173,7 +173,8 @@ def __init__(
         self.object_lookup = {o["featureNodeHash"]: o for o in self.project.ontology["objects"]}
 
         logger.info("Fetching project label rows to be able to match predictions.")
-        self.label_rows = prepare_data(cache_dir, project=project, **kwargs).label_rows
+        # todo check if it is really necessary to read from project from Encord (probably should be local storage)
+        self.label_rows = Project.from_encord_project(cache_dir, project).label_rows
```

Comment:
pretty sure that is correct since to import predictions we need a local project

Reply:
I 100% agree.